packages/protocol-contracts/test/ZetaConnector.spec.ts

```diff
@@ -671,5 +674,64 @@ describe("ZetaConnector tests", () => {
         expect(e2.length).to.equal(1);
       });
     });
+
+    describe("MaxSupply", () => {
+      it("Should revert if the updater is not the TSS updater", async () => {
+        await expect(zetaConnectorNonEthContract.connect(randomSigner).setMaxSupply(0)).to.revertedWith(
+          `CallerIsNotTss("${randomSigner.address}")`
+        );
+      });
+
+      it("Should revert if want to mint more than MaxSupply", async () => {
+        await zetaConnectorNonEthContract.connect(tssSigner).setMaxSupply(999);
+        await expect(
+          zetaConnectorNonEthContract
+            .connect(tssSigner)
+            .onReceive(
+              randomSigner.address,
+              1,
+              zetaReceiverMockContract.address,
+              1000,
+              new ethers.utils.AbiCoder().encode(["string"], ["hello"]),
+              "0x0000000000000000000000000000000000000000000000000000000000000000"
+            )
+        ).to.revertedWith(`ValueAboveMaxSupply(999)`);
+      });
+
+      it("Should mint it's value is bellow MaxSupply", async () => {
```

Comment:
```suggestion
      it("Should mint it's value is below MaxSupply", async () => {
```

Reply:
please let's do a self review on Github itself before sending the PR to avoid as many typos as we can